src/components/Header.tsx

```diff
@@ -185,11 +200,11 @@ export function Header({
               <select
                 className="jdl-select"
                 value={jhOnline.jdlId}
-                onChange={setJDL}
+                onChange={(e) => setJDL(e.target.value)}
```

Comment:
lets do a const arrow function outside the event for this, there is a minor performance benefit in that (very minor but I'm OCD about these :wink: )

Reply:
Aha no problem, it's fixed !